scenarioo-client/test/e2e/specs/step_view.ts

```diff
@@ -117,7 +118,8 @@ useCase('Step - View')
 
         scenario('Metadata with link to object')
             .description('Click on a object link in Call tree and jump to object example.action.StartInitAction')
-            .it(async () => {
+            //TODO #946 activate this test once expand collapse of metadata works
+            .xit(async () => {
```

Comment:
@eggerschwiler anstatt xit bitetet scenariooJS für länger unterdrückte tests ein spezielles "pending" wo man einen gelogten kommentar angeben kann - xit sollte man nie nie nie commiten, dafür wollten wir mal einen precommit hook machen.

Reply:
dies ist analog zu jasmine pending für solche fälle - iseh zB zweite Antwort hier: https://stackoverflow.com/questions/11518609/declare-pending-specs-tests-in-jasmine-or-mocha

Reply:
Thanks for the hint. I wasn't aware of the "pending" option. Updated the tests accordingly.